TRACING.md

```diff
@@ -0,0 +1,312 @@
+# Agent Tracing
+
+* [Introduction](#introduction)
+* [OpenTracing summary](#opentracing-summary)
+* [Jaeger tracing architecture](#jaeger-tracing-architecture)
+* [VM shutdown](#vm-shutdown)
+* [Adding tracing support to the agent](#adding-tracing-support-to-the-agent)
+* [Guest to Host communication using VSOCK](#guest-to-host-communication-using-vsock)
+* [Enabling tracing](#enabling-tracing)
+  * [Tracing options](#tracing-options)
+* [Running the agent with tracing enabled](#running-the-agent-with-tracing-enabled)
+* [Requirements](#requirements)
+  * [Host environment](#host-environment)
+  * [Guest environment](#guest-environment)
+* [Limitations](#limitations)
+* [Appendix: Setting up a tracing development environment](#appendix:-setting-up-a-tracing-development-environment)
```

Comment:
The appendix link does not hot-render. I think it needs the ref modding to (that is, drop the stray `:`) :
```
#appendix-setting-up-a-tracing-development-environment
```

Reply:
Fixed.